### ISSUE TITLE
embiggen broken on development/mps

### ISSUE THREAD

**Issue by krummrey** [invoke-ai/InvokeAI#570]
**Describe your environment**
- GPU: [mps]
- CPU arch: [arm]
- OS: [macOS]
- Python: [miniconda]
- Branch: [development]

**Describe the bug**
Using embiggen breaks. I was trying to reproduce the images from the embiggen doc and it failed

**To Reproduce**
```
dream> a photo of a forest at sunset -s 100 -W 512 -H 512 -I outputs/forest.png -f 0.4 -embiggen 2
>> loaded input image of size 512x512 from outputs/forest.png
>> ESRGAN upscaling init image prior to cutting with Embiggen with strength 0.75
>> Real-ESRGAN Upscaling seed:None : scale:2x
	Tile 1/4
	Tile 2/4
	Tile 3/4
	Tile 4/4
Generating:   0%|                                                                                                                               | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/Users/jan/Documents/ML/stable-diffusion/ldm/generate.py", line 324, in prompt2image
    results = generator.generate(
  File "/Users/jan/Documents/ML/stable-diffusion/ldm/dream/generator/base.py", line 67, in generate
    x_T = self.get_noise(width,height)
  File "/Users/jan/Documents/ML/stable-diffusion/ldm/dream/generator/base.py", line 116, in get_noise
    raise NotImplementedError("get_noise() must be implemented in a descendent class")
NotImplementedError: get_noise() must be implemented in a descendent class

>> Could not generate image.
>> Usage stats:
>>   0 image(s) generated in 17.06s
Outputs:
```

**Comment by lstein** [comment]
Check out pull request #553 . It should fix your problem. It is not quite ready to pull into development or main at this point, but will be soon.

**Comment by psychedelicious** [comment]
This was resolved by #553.
